events/reporter.go

```diff
@@ -467,8 +471,8 @@ func UnsubscribeFromReceipts(subscriptionChannel chan TxReceipt) {
 
 // InitializeEventReporter initializes the event reporting interface
 func InitializeEventReporter(url string) error {
-	// By default use zero-buffer channels and non-blocking.
-	return InitializeEventReporterWithOptions(url, 0)
+	// Need to buffer the channels so that slow subscribers can still receive updates
+	return InitializeEventReporterWithOptions(url, 30)
```

Comment:
please make this a parameter rather than hard-coding the number

Reply:
Should I add this to the `Config` struct?